lms/product/moodle/_plugin/misc.py

```diff
@@ -0,0 +1,11 @@
+from lms.product.plugin.misc import MiscPlugin
+
+
+class MoodleMiscPlugin(MiscPlugin):
+    deep_linking_prompt_for_title = False
```

Comment:
I've been suffering this while testing, fixed it on this PR.

The main issue is while editing an existing assignment using Moodle we prompt for a title again, use the "Hypothesis Assignment" default and we overwrite the title of the assignment.